src/translation/translation_nb_NO.ts

```diff
@@ -717,117 +717,117 @@
         <location filename="../clientdlg.cpp" line="229"/>
         <location filename="../clientdlg.cpp" line="1263"/>
         <source>C&amp;onnect</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Koble til</translation>
     </message>
     <message>
         <location filename="../clientdlg.cpp" line="282"/>
         <source>&amp;File</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Fil</translation>
     </message>
     <message>
         <location filename="../clientdlg.cpp" line="284"/>
         <source>&amp;Connection Setup...</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Tilkoblingsoppsett …</translation>
     </message>
     <message>
         <location filename="../clientdlg.cpp" line="288"/>
         <source>&amp;Load Mixer Channels Setup...</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">&amp;Last inn kanalmikserinnstillingene …</translation>
```

Comment:
Good point about the typography here. Should the source string should also use `…` instead of three dots?

Reply:
Using special typography should, in my view, be avoided.  I don't know how a screen reader would render the special character, for example.  Using `...` to indicate a menu item opens a new window is pretty standard, too - not following that might also cause accessibility issues.  I think it doesn't look good, either.

Reply:
@pljones
The error here is the missing hyphen(s), and/or prepositions.
The translation is ~"Load the settings for the channel mixer", which I got from looking at the Swedish string.

The use of an ellipsis is correct, insofar as both typography and current paradigms in software go.
A screen reader will not output "dot" "dot" "dot", which is annoying and wastes time.
The ellipsis is the correct character to use, and Weblate triggers and error when this isn't the case.
If it doesn't look good for you, it is possible to change the font used.

https://develop.kde.org/hig/style/writing/placeholder/
https://developer.gnome.org/hig/guidelines/typography.html
https://wiki.amigaos.net/wiki/UI_Style_Guide_Menus#Ellipsis
https://www.stylemanual.gov.au/grammar-punctuation-and-conventions/punctuation-and-capitalisation/ellipses#do_not_use_a_string_of_full_stops
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/ellipsis-button

The ellipsis does not create issues unless UTF-8 support is missing.